server/protocols/jmap/src/test/java/org/apache/james/jmap/model/MessageTest.java

```diff
@@ -89,18 +90,18 @@ public void buildShouldThrowWhenDateIsNull() {
     @Test(expected=IllegalStateException.class)
     public void buildShouldThrowWhenPreviewIsNull() {
         Message.builder().id(TestMessageId.of(1)).blobId(BlobId.of("blobId")).threadId("threadId").fluentMailboxIds().headers(ImmutableMap.of())
-            .subject("subject").size(123).date(ZonedDateTime.now()).build();
+            .subject("subject").size(123).date(Instant.now()).build();
```

Comment:
You should not call XXX.now() in your tests. Please prefer stable, always the same constant